test-data/unit/check-classes.test

```diff
@@ -2051,6 +2051,16 @@ class X: pass
 foo(X)
 [out]
 
+[case testTypeUsingTypeCTypeAnyMember]
+from typing import Type, Any
+def foo(arg: Type[Any]):
+    x = arg.member_name
```

Comment:
That is because outside the test suite it uses type.mro from typeshed, which is defined as `def mro(self) -> List[type]: ...`. Presumably the test fixtures don't have that method defined.